src/public/js/util.js

```diff
@@ -38,6 +38,12 @@ var UTIL = (function() {
      0xb4, 0x7c, 0xb8, 0x26, 0x77, 0x99, 0xe3, 0xa5,
      0x67, 0x4a, 0xed, 0xde, 0xc5, 0x31, 0xfe, 0x18,
      0x0d, 0x63, 0x8c, 0x80, 0xc0, 0xf7, 0x70, 0x07]);
+  log3.get = function(x) {
```

Comment:
So I want to make sure I am understanding what is happening here correctly.

You are creating a new field of the object log3 which holds a Uint8Array right now which is called get and putting in that field a function(function pointer really but you know what I mean) that does the getting and returns the element or an error.

So my understanding here is that you can dynamically add fields to objects in JS(since the docs for Array and typed Array seem to suggest it doesn't have a field called get)? That is pretty cool. Is there a way to check if an object has a field? Like what happens if i ask for log3.has but never gave log3 a has field?